### ISSUE TITLE
Trying to hack my Apple TV 1Gen. Need help…

### ISSUE THREAD

**Issue by Agelandkaai** [davilla/atvusb-creator#646]
Hi, i found atvusb-creator-1.0.b13, but it gives an error when I try to open it.

An unexpected error has occured during execution of the main script. 
OSError: [Errno 30] Read-only file system: '/private/var/folders/2I/2_jg_h7n5kndwxqwyk4dgf7m0000gn/T/AppTranslocation/BD2D7F55-0930-47BB-94D9-6A279B395CAA/d/staging'

My iMac runs 10.13.2

I bought a 1gen Apple TV to use it as a hard disk, since I have no wifi in my laundryroom. 
We want to watch some tv while ironing…

If someone can help us, we would be very gratefull.
Also maybe a nice tutorial? Not really good at Terminal stuff, copy and paste I can do, but don't want to mess up my iMac.

Thank you

**Comment by amet** [comment]
https://github.com/davilla/atvusb-creator/releases/tag/1.0.b13

You need to run sudo command as per the note on the release. That’s the only way it can be done unfortunately :(

**Comment by Agelandkaai** [comment]
Is that then to make the patchstick?
btw, thank you for ur swift response.

**Comment by Agelandkaai** [comment]
Thank you very much, got it to open.
For other people who are (like me) a bit afraid to use the terminal,
I opened it with Show Package, after left clicking on the file.
Then clicking Contents, Mac OS --> ATV usb creator.

**Comment by amet** [comment]
Good to know, let us know if you run into any other issues

**Comment by Agelandkaai** [comment]
yeah, well, the program keeps saying there is no usb drive to be found.

**Comment by amet** [comment]
How big is the usb stick and can you paste the output from terminal when you run the sudo

**Comment by Agelandkaai** [comment]
128 gb

**Comment by Agelandkaai** [comment]
on my laptop now, can paste it later

**Comment by amet** [comment]
128 should work, we accept all up to 256.

**Comment by Agelandkaai** [comment]
Worked with an old ipod shuffle 1GB.
How do I get to watch movies from on the hard drive. without needing wifi?

**Comment by Agelandkaai** [comment]
Spoke too soon
![Schermafbeelding 2019-03-09 om 13 10 00](https://user-images.githubusercontent.com/48350443/54071438-c6293000-426c-11e9-8bd1-15f94fb1476f.png)

**Comment by amet** [comment]
its important to run the command I linked you to, it has to run as sudo for it to be able to create the  patchstick

```sudo /atvusb-creator.app/Contents/MacOS/atvusb-creator```

```e.g: MacBook-Pro-3:~ amet$ sudo /Users/amet/Desktop/atvusb-creator.app/Contents/MacOS/atvusb-creator```

**Comment by Agelandkaai** [comment]
So I open the terminal, and just copy the command in to it?
I'll try

**Comment by Agelandkaai** [comment]
Command not found
Where does the file need to be? Now its in Downloads.

**Comment by amet** [comment]
not really :)

type sudo and "space" ,  find atvusb-creator.app Show contents and find atvusb-creator, you can drag/drop atvusb-creator into terminal window.. it would look like below 

```sudo /Users/amet/Desktop/atvusb-creator.app/Contents/MacOS/atvusb-creator```

**Comment by Agelandkaai** [comment]
![Schermafbeelding 2019-03-09 om 13 26 36](https://user-images.githubusercontent.com/48350443/54071596-fa055500-426e-11e9-85ff-18817d56efbe.png)
So i typed sudo & space, then dropped the app on the terminal and then pressed Enter, got this

**Comment by Agelandkaai** [comment]
after typing my password, that is

**Comment by amet** [comment]
you need to read what I posted :)
```sudo /Users/iMac/Downloads/atvusb-creator.app/Contents/MacOS/atvusb-creator```

**Comment by Agelandkaai** [comment]
okay, dropped the app from the package, got this far
now running the script again

**Comment by Agelandkaai** [comment]
Lasting longer already now

**Comment by Agelandkaai** [comment]
copying files to target disk

**Comment by Agelandkaai** [comment]
copy files to target disk
  remove recovery hfsplus GUID
  umount_disk settling delay (10 seconds)
  change recovery to real GUID
Unable to add real recovery: gpt add: unable to open device '/dev/disk3': No such file or directory
  install_recovery failed
Failed! (0:04:13)

**Comment by Agelandkaai** [comment]
don't see the usb in my Finder sidewindow, or with disk utility

**Comment by Agelandkaai** [comment]
the ipod is still flashing it's little orange light

**Comment by amet** [comment]
You will have to find another usb to use, I am not sure it will work with an iPod

**Comment by Agelandkaai** [comment]
the only other one i have is the 128, which he doesn't see, or an even older ipod shuffle, the "long" white one, this is the silver one with a clip

**Comment by amet** [comment]
Patchstick creator needs to partition usb into 2 separate partitions, not sure that’s possible with iPod ... I have not tried it so can’t offer any advice on it

**Comment by Agelandkaai** [comment]
With the 128 Gb, he keeps saying 
Unable to find any USB drives

**Comment by Agelandkaai** [comment]
And may I ask, what does Bootloader do?

**Comment by amet** [comment]
It’s all part of “jailbreaking” the atv1 and opening it up to be seen as a storage drive and media player.

How you planning to watch movies from it? Not much software will run on it

**Comment by Agelandkaai** [comment]
I actually just want to use the hd, where i would store the files and want to watch these, since we don't have wifi in the laundryroom.

**Comment by amet** [comment]
Usb you are trying to create will only open atv1 for you to use terminal and add software to it, it will not have any additional software in it for you to watch anything

**Comment by Agelandkaai** [comment]
But it's possible to add software, to watch files from the hard drive?

Think I still have a usb stick on my sailing boat that stores music.
Maybe I'll go and get it later today.

**Comment by Agelandkaai** [comment]
Is it difficult to add the software? Otherwise I bought this one with no reason…

**Comment by Agelandkaai** [comment]
Fingers crossed, went for a normal usb-stick on the boat.

**Comment by Agelandkaai** [comment]
Okay, have the patch-stick.

**Comment by Agelandkaai** [comment]
Is it normal the apple tv doesn't show up in my iTunes? Have iTunes 12.8.…

**Comment by Agelandkaai** [comment]
Airplay works, but don't get a list of devices

**Comment by amet** [comment]
Yes it normal, Apple dropped support for it long time ago

Nothing we worked on in 2009-10 still runs on that hardware :(

As far as I can remember, osmc.tv is the only player solution , not sure how it works or if you can get movies on it.

https://osmc.tv/wiki/apple-tv/frequently-asked-questions/

**Comment by amet** [comment]
Last time I used atv1 was approximately 9 years ago, since then many things have changed.

Remember that if you install osmc it will delete all content from that appletv and instal Linux on it, just a warning :)

**Comment by Agelandkaai** [comment]
So now I restarted the Apple tv, with the usb in it.
It says if no errors, plug in your aple tv, wait 3 minutes, and open Tewrminal.
How do i plug my apple tv in to my iMac? ethernet?

**Comment by amet** [comment]
Apple TV needs to be on Wifi or on LAN

But I still fail to see what you will do after you successfully ran a patchstick on it... it will give you a terminal access and a new menu... that’s it

Menu is no good as all the apps that were in it are long gone

**Comment by Agelandkaai** [comment]
He's on the same wifi as my iMac.
Typed ssh frontrow@appletv.local  enter
nothing happened

So I can't then just use the HD to store files, and play them without using iTunes?
That was the reason I bought this one.

**Comment by Agelandkaai** [comment]
And I can't use it anymore either with iTunes?
Wath was the last supported version of iTunes?

**Comment by amet** [comment]
> He's on the same wifi as my iMac.
> Typed ssh [frontrow@appletv.local](mailto:frontrow@appletv.local) enter
> nothing happened
> 
> So I can't then just use the HD to store files, and play them without using iTunes?
> That was the reason I bought this one.

no, there is no way for you to use the HD to store files and play them using ATV1 player

**Comment by amet** [comment]
> And I can't use it anymore either with iTunes?
> Wath was the last supported version of iTunes?

not sure what version was last that supported it, if I had to guess.. 5 years ago or so

**Comment by Agelandkaai** [comment]
Anyway, a very big Thank You for what you've helped me with already.

**Comment by amet** [comment]
👍
